pom.xml

```diff
@@ -88,6 +100,23 @@
       <optional>true</optional>
     </dependency>
 
+    <dependency>
+      <groupId>org.postgresql</groupId>
```

Comment:
Why do we need postgres?

Reply:
We will need it for the further story - https://issues.folio.org/browse/MSEARCH-9